.github/workflows/shell.yml

```diff
@@ -0,0 +1,21 @@
+name: 🐚 Lint Shell
+
+on:
+  workflow_dispatch: {}
+  push:
+    paths:
+    - "**.sh"
+
+jobs:
+  lint-shell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Shellcheck and shformat
+        uses: luizm/action-sh-checker@master
+        with:
+          sh_checker_exclude: "third_party/"
+          sh_checker_comment: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'm probably missing something, but why do we need the `GITHUB_TOKEN` here?

Reply:
Shellcheck wants it in order to comment on the diff, but I don't think that is useful actually, since we don't have anything else that comments on diffs and usually check the Action output. I'll remove it (and the related `sh_checker_comment: true`).